From 8b418699dacae8cdab1d5b72fdf9b1bcf31f6133 Mon Sep 17 00:00:00 2001
From: Alec Leamas <alec@tests.notat.diaspora.com>
Date: Tue, 30 Oct 2012 20:38:19 +0100
Subject: [PATCH] Dont configure libgsm

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 9117695..d141104 100644
--- a/configure.ac
+++ b/configure.ac
@@ -603,7 +603,7 @@ AC_SUBST(PLAYRECLINKS)
 AM_CONDITIONAL(STATIC_LIBSOX_ONLY, test "$enable_shared" = "no" -a "$enable_static" = "yes")
 
 dnl Generate output files.
-AC_CONFIG_FILES(Makefile src/Makefile libgsm/Makefile lpc10/Makefile msvc9/Makefile msvc10/Makefile sox.pc)
+AC_CONFIG_FILES(Makefile src/Makefile lpc10/Makefile msvc9/Makefile msvc10/Makefile sox.pc)
 AC_OUTPUT
 
 if test "$using_gsm" != "no"; then
-- 
1.7.11.4

